tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_homescreen_change_wallpaper.py

```diff
@@ -18,7 +20,8 @@ def test_homescreen_change_wallpaper_from_gallery(self):
         https://moztrap.mozilla.org/manage/case/1902/
         reusing /functional/homescreen/test_homescreen_change_wallpaper.py script
         """
-
+        # wait until the homescreen is fully drawn
+        time.sleep(3)
```

Comment:
Yeah, I don't see anything obvious to detect this, either. Perhaps it would be possible to check for the homescreen height to have a certain threshhold or something? But that's rather hacky, too.